src/profile-logic/tracks.js

```diff
@@ -627,6 +627,13 @@ function _isThreadIdle(profile: Profile, thread: Thread): boolean {
     );
   }
 
+  if (/^(?:Audio|Media)/.test(thread.name)) {
```

Comment:
Could you please do some "e.g." examples to help document the intent of this regex? Also, TIL non-capturing groups in regex.

Reply:
did it, thanks for the good suggestion!